org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -10,6 +10,19 @@ var OpenT2T = require('opent2t').OpenT2T;
 var Crypto = require('crypto');
 
 /**
+ * Gets a property from a JSON dictionary without case sensitivity
+ * this will return the first item it finds, though multiple may match
+ * in JSON.
+ */
+function getDictionaryItemCaseInsensitive(obj, propertyName) {
+    for(var name in obj) {
+        if (propertyName.toUpperCase() == name.toUpperCase()) {
```

Comment:
Can't use hasOwnProperty as it is case sensitive, I need to loop through all the properties and compare.

---
In reply to: [101857303](https://github.com/openT2T/translators/pull/166#discussion_r101857303) [](ancestors = 101857303)